### ISSUE TITLE
Proxy Auth

### ISSUE THREAD

**Issue by iAweX** [microsoft/vscode#104726]
Issue Type: <b>Bug</b>

Proxy Auth does not work after update, when using system proxy on Windows

VS Code version: Code 1.48.0 (db40434f562994116e5b21c24015a2e40b2504e6, 2020-08-13T07:50:42.600Z)
OS version: Windows_NT x64 10.0.14393

**Comment by gjsjohnmurray** [comment]
Probably #104603 which offers a workaround.

**Comment by MartinMajewski** [comment]
OS version: macOS 10.15.6 (19G73)
VSC version: 1.48.0

I also have Proxy Auth issues with the popup dialog.

When I start VSC while connected to my corp network, VSC asks me for proxy auth (as usual). However, the dialog does not react to "OK" nor "Cancel." As it is blocking and without a window header, I cannot use VSC anymore.

![Screenshot 2020-08-17 at 09 18 50](https://user-images.githubusercontent.com/2221289/90369319-e2d5b400-e06b-11ea-861f-bd1861886a86.png)


While reporting this issue here, I used the short amount of time between VSC startup and the proxy auth dialog pop up to quickly open the About page of VSC to get the version number of VSC.

I discovered that while the About page is open the proxy auth dialog will appear as full dialog window with the "close window" icon in the left corner. While the dialog still does not react to "OK", "Close", "Enter" nor "Escape", one can close the window with the header icon. The popup will show once more but can be closed the same way and VSC is usable again.

![Screenshot 2020-08-17 at 09 17 42](https://user-images.githubusercontent.com/2221289/90369340-e8cb9500-e06b-11ea-948c-ba982e0d6ddc.png)
